src/pages/Repo/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {BackButton} from '../../components'
+
+function Repo({selectedRepo}) {
+    const date = new Date(selectedRepo.updated_at)
+    return(
+        <>
+        <BackButton />
+        <div id="repo-display">
+        <h1>{selectedRepo.name}</h1>
+        {selectedRepo.description && <h3>{selectedRepo.description}</h3>}
+        <ul>
+            <li>Issue count: {selectedRepo.open_issues_count}</li>
+            <li>Forks: {selectedRepo.forks_count}</li>
```

Comment:
Could add some of the basics here (like the description) to the list on the previous page, also could've added a link, but other than that all good

Reply:
Completely right, we were going to put a link but just forgot about it. What do you mean about adding the description on the previous page? what would you include on this page?

Reply:
Like just adding the description to the list of repos as well, so that when you're searching you  can have a bit more of a reminder about what the repo was